### ISSUE TITLE
AriaLabel changes back to original state when accessibility mode is turned off

### ISSUE THREAD

**Issue by Sembauke** [microsoft/monaco-editor#2587]
1. Try to reproduce in VS Code and file issue directly at:
- Write a function that checks if the editorMounted 
- Write a `editor.updateOptions({ ariaLabel: 'Text'})`
- Turn on accessibility mode 
- open dev tools in chrome or another browser
- press ctrl + F search for textarea 
- See current aria-label 'text' 
- Turn off accessibilitymode
- Label changes back to the original message

**monaco-editor version:**
 0.26.1

**Browser:**
Chrome
**OS:**
Windows

**Comment by alexdima** [comment]
This is intentional. When accessibility mode is off, the editor is not accessible. Therefore, it uses its aria label to instruct Screen Reader users on how to turn on accessibility mode.
